<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImageToPDFDialog</class>
 <widget class="QDialog" name="ImageToPDFDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1015</width>
    <height>725</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ImageToPDF</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>40</y>
     <width>141</width>
     <height>61</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Apply|QDialogButtonBox::Close</set>
   </property>
   <property name="centerButtons">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>641</width>
     <height>501</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="MainTab">
    <attribute name="title">
     <string>Menu</string>
    </attribute>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="plusButton">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../Images/res.qrc">
           <normaloff>:/newPrefix/plus.jpg</normaloff>:/newPrefix/plus.jpg</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="minusButton">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../Images/res.qrc">
           <normaloff>:/newPrefix/minus.jpg</normaloff>:/newPrefix/minus.jpg</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="removeAllButton">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../Images/res.qrc">
           <normaloff>:/newPrefix/cancel.jpg</normaloff>:/newPrefix/cancel.jpg</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="3" column="1" rowspan="2" colspan="2">
      <widget class="Line" name="line">
       <property name="lineWidth">
        <number>3</number>
       </property>
       <property name="midLineWidth">
        <number>3</number>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="1" rowspan="2" colspan="2">
      <widget class="Line" name="line_3">
       <property name="lineWidth">
        <number>3</number>
       </property>
       <property name="midLineWidth">
        <number>3</number>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item row="2" column="0" rowspan="2" colspan="4">
      <widget class="Line" name="line_2">
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <property name="lineWidth">
        <number>3</number>
       </property>
       <property name="midLineWidth">
        <number>3</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QComboBox" name="comboBox">
         <property name="editable">
          <bool>false</bool>
         </property>
         <property name="currentText">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="4" column="0">
      <widget class="QTableWidget" name="InputFiles">
       <column>
        <property name="text">
         <string>Name</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Rotate</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Resize</string>
        </property>
       </column>
      </widget>
     </item>
     <item row="4" column="3">
      <widget class="QListWidget" name="OutputFiles">
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="lineWidth">
        <number>3</number>
       </property>
       <property name="midLineWidth">
        <number>1</number>
       </property>
       <property name="autoScrollMargin">
        <number>10</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="SettingsTab">
    <attribute name="title">
     <string>Settings</string>
    </attribute>
   </widget>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>677</x>
     <y>140</y>
     <width>121</width>
     <height>61</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>PushButton</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_4">
      <property name="text">
       <string>PushButton</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="ImagePreview">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>260</y>
     <width>231</width>
     <height>181</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../Images/res.qrc"/>
 </resources>
 <connections/>
</ui>
